Source/WebCore/xml/XSLImportRule.cpp

```diff
@@ -53,11 +51,9 @@ void XSLImportRule::setXSLStyleSheet(const String& href, const URL& baseURL, con
     if (m_styleSheet)
         m_styleSheet->setParentStyleSheet(nullptr);
 
-    m_styleSheet = XSLStyleSheet::create(this, href, baseURL);
-
-    XSLStyleSheet* parent = parentStyleSheet();
-    if (parent)
-        m_styleSheet->setParentStyleSheet(parent);
+    // FIXME: parentStyleSheet() should never be null here.
```

Comment:
I don’t understand why this comment says `FIXME`. What are we going to fix?

Reply:
Ideally, we pass a reference instead of a pointer if it's never null. I can't easily validate that, however.